android/manifest.xml

```diff
@@ -67,15 +69,6 @@
   </intent-filter>
 </receiver>
 
-<receiver android:name="com.onesignal.GcmBroadcastReceiver"
```

Comment:
Is it okay to remove these ?
Has this plugin migrated to FCM ?

Reply:
Onesignal has not moved to FCM still. We should keep play services settings

Reply:
Above settings are not required as I see in OneSignal manuals:
https://documentation.onesignal.com/docs/android-sdk-setup
I will remove them and test how it works with OneSignal.